routes/artifacts.js

```diff
@@ -361,13 +368,18 @@ var replyWithArtifact = async function(taskId, runId, name, req, res) {
   if(artifact.storageType === 's3') {
     // Find url
     var url = null;
-    var prefix = artifact.details.prefix;
-    if (artifact.details.bucket === this.publicBucket.bucket) {
-      var region = this.regionResolver.getRegion(req);
-      if (region) {
-        url = 'http://' + this.publicProxies[region] + '/' + prefix;
-      } else {
-        url = this.publicBucket.createGetUrl(prefix);
+    
+    if (task.tags.useCloudMirror) {
```

Comment:
yeah, task.extra is a good option...

----
For the record I fully support just moving over everything and once and see the world burn... It's much faster to revert if we see issues...